packages/driver/src/cypress/runner.ts

```diff
@@ -1781,7 +1776,16 @@ export default {
           _testsQueue.push(test)
         }
 
-        const existing = _logsById[attrs.id]
+        const { instrument } = attrs
+
+        // test = getTestById(testId)
+
+        // if (test) {
+        // pluralize the instrument as a property on the runnable
```

Comment:
What's an example of 'instrument' that we're pluralizing?

Reply:
This could be command, route, hook or agent. Sessions is the odd ball instrument to date and is tied to command